fastify.js

```diff
@@ -674,6 +674,17 @@ function build (options) {
     return _fastify
   }
 
+  function defaultErrorHandler (error, request, reply) {
+    var res = reply.res
+    var statusCode = res.statusCode ? res.statusCode : 500
```

Comment:
This seems unnecessary since `res.statusCode` should always be valid here (it gets set [here](https://github.com/fastify/fastify/pull/1135/files#diff-d2fdf63c1d7f1d9f68270b6dfa8c930cR310) just before the error handler is called).